ihatemoney/tests/tests.py

```diff
@@ -77,7 +78,7 @@ def create_project(self, name):
 
 
 class IhatemoneyTestCase(BaseTestCase):
-    SQLALCHEMY_DATABASE_URI = "sqlite://"
+    SQLALCHEMY_DATABASE_URI = "sqlite:////tmp/ihatemoneytest.db"
```

Comment:
```suggestion
    SQLALCHEMY_DATABASE_URI = "sqlite://"
```
Revert Mistakenly Committed Debugging Change